src/convertToObject.js

```diff
@@ -6,7 +6,35 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // This onbject will be returned.
+  const styledObject = {};
+
+  // Who the hell write sourceString like that? My grandma can do better...
+  // If i ever see this during actuall job, i'll leave the company.
+  //                                         (through window)
+  const cleanString = sourceString
+    .replace(/\s*;\s*$/, '')
+    .replace(/\s*:\s*/g, ':')
+    .trim();
+
+  // Split into elements.
+  const splitToElements = cleanString.split(';').filter(Boolean);
```

Comment:
[CODE STYLE] All chained methods called on the same level should have same indentation. This makes code much more readable. You should write it in the following way: `cleanString.split(';')
.filter(Boolean);`